.github/workflows/distribute-grain.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: |
           echo ::set-env name=PULL_REQUEST_TITLE::"Scheduled grain distribution for week ending $(date +"%B %dth, %Y")"
           echo ::set-env name=PULL_REQUEST_BODY::"This PR was auto-generated on $(date +%d-%m-%Y) \
-            to add the latest grain distribution to our instance. $(cat grain_output.txt)"
+            to add the latest grain distribution to our instance. <br /> <br />  $(cat grain_output.txt)"
```

Comment:
Putting html in the PR body seems weird--when this gets converted into a commit message, will it contain literal html? Can we just add double newlines?

Reply:
@decentralion I've tried subtler ways to created new lines in the markdown string like `\n` and raw carriage returns in the yaml file. We can get away with one raw carriage return but beyond that any existing text is truncated.

Newline escapes just render back as like they do here: \n

Reply:
I can add a *** to create an `<hr />` element if that suits you better. Will render the grain logout a a definitive newline and won't look offensive in a commit message

Reply:
I have one more idea to test

Reply:
I've confirmed that any way to hack two consecutive raw carriage returns into the string using bash causes the string to be truncated by github. I'll add the <hr> element for now